core/o11y/benches/metrics.rs

```diff
@@ -63,7 +63,7 @@ fn inc_counter_vec_with_label_values_stack_no_format(bench: &mut Bencher) {
             loop {
                 idx -= 1;
                 buf[idx] = b'0' + (n % 10) as u8;
-                n = n / 10;
+                n /= 10;
```

Comment:
this also feels a bit odd to me, eitherway it's fine but personally I don't think `n /= 10`  is better than `n = n / 10`